src/Sentry.Maui/SentryMauiOptions.cs

```diff
@@ -56,4 +56,12 @@ public SentryMauiOptions()
     /// Use caution when enabling, as depending on your application this might incur a performance overhead.
     /// </summary>
     public bool CreateElementEventsBreadcrumbs { get; set; } = false;
+
+    /// <summary>
+    /// Automatically attaches a screenshot of the app at the time of the event capture.
+    /// </summary>
+    /// <remarks>
+    /// Be aware PII might be included by activating this feature.
+    /// </remarks>
+    public bool AttachScreenshots { get; set; }
```

Comment:
```suggestion
    public bool AttachScreenshot { get; set; }
```

The property name should be `AttachScreenshot`. As it's on the native iOS/Android.

Also please make sure to pass the value to the native layer in both platforms (prob MacCatalyst too) since that controls a screenshot if a native crash happens, for example. 

That also means making sure we don't create two screenshots in some weird situation.